index.html

```diff
@@ -6,8 +6,79 @@
   <meta name="Joshua Serpis" content="name">
```

Comment:
Change variable names to make them more appropriate and look at using a table for the format